calico_cni/calico_cni.py

```diff
@@ -233,7 +242,11 @@ def delete(self):
 
         # Step 3: Delete the veth interface for this endpoint.
         _log.info("Removing veth for endpoint: %s", endpoint.name)
-        netns.remove_veth(endpoint.name)
+        try:
+            if not netns.remove_veth(endpoint.name):
+                _log.warning("Veth for host %s does not exist.", endpoint.name)
```

Comment:
Change the wording here - host implies the VM or metal that the container is running on, when the veth itself is actually a per-container / pod construct.

```
"Veth %s does not exist", endpoint.name
```
and 
```
"Unable to remove veth %s", endpoint.name
```